package.json

```diff
@@ -45,6 +46,7 @@
     "packages:lint": "lerna run lint --parallel --no-sort",
     "packages:start": "lerna run start --parallel",
     "packages:test": "lerna run test",
+    "packages:test:unit": "lerna run test:unit --ignore '@lmc-eu/spirit-web-twig'",
```

Comment:
why is it ignored?

Reply:
Good point. Because of the usage in GitHub Actions workflow. But I have done this better. Fixed.